src/index.js

```diff
@@ -4,17 +4,19 @@ import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
 import {BrowserRouter} from "react-router-dom";
-import { CategoryProvider } from "./Context";
+import { CategoryProvider, VideoProvider } from "./Context";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
+    <VideoProvider>
     <CategoryProvider>
     <App />
     </CategoryProvider>
+    </VideoProvider>
```

Comment:
And instead of adding the providers in index.js you can create a new file called as CombinedProviders.jsx

```
import React from "react";
import { VideoProvider, CategoryProvider } from "LOCATION WHERE CONTEXTS ARE STORED";

const CombinedProvider = ({ children }) => {
    return (
        <VideoProvider>
                <CategoryProvider>
                {children}
                </CategoryProvider>
            </VideoProvider>
    )
}

export { CombinedProvider };
```

And then you can start using it in your index.js file just shifting your work from one file to another

Reply:
oh...this is new

Reply:
good idea